### PR TITLE
spoof people

### PR DESCRIPTION
**Pull request by forkfork**
This allows the middleware to inject req.user.spoofedBy when a user is spoofing another user.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -127,6 +127,9 @@ function verifyUser(options){
             }
           })
           .then(async function (data) {
+            if (data) {
+              data.spoofedBy = req.jwt.aud;
```
Comment (ludbek) [inline index.js]: Noice :D

Comment (forkfork) [reply index.js]: Suren!

Comment (colins44) [reply index.js]: @ludbek are you spoffing us?

Comment (forkfork) [reply index.js]: @ludbek

Comment (ludbek) [reply index.js]: Oh yeah, thats why I am keeping close eyes on these public repos hehe
Btw I have a copy of all the valid `jwt` tokens from our auth service :D (kidding...)

**Pull request merged**